check_process

```diff
@@ -20,13 +20,9 @@
 		setup_private=0
 		setup_public=0
 		upgrade=1
-		upgrade=1	from_commit=1cdeb9f003f07c4df897f29dd4a521c595327415
```

Comment:
Maybe you should keep an upgrade test from a previous version

Reply:
We cannot anymore, sadly: previous versions are not installable anymore because of what contained a dependency `package.js`. We get this error:
https://gitlab.com/mx-puppet/discord/mx-puppet-discord/-/issues/213